spec/unit/pops/parser/lexer2_spec.rb

```diff
@@ -551,7 +551,39 @@ def expect_issue(code, issue)
       [:VARIABLE, "x"],
       :EQUALS,
       [:NUMBER, "10"],
-      [:RENDER_STRING, "just text\n"]
+      [:RENDER_STRING, "      just text\n"]
+      )
+    end
+
+    it 'epp comments strips left whitespace when preceding is right trim' do
+      code = <<-CODE
+      This is <% $x=10 -%>
+      space-before-me-but-not-after   <%# This is an epp comment %>
+      just text
+      CODE
+      epp_tokens_scanned_from(code).should match_tokens2(
```

Comment:
Please use 'expect' syntax and not 'should' on new tests as the latter creates deprecation warnings when merged to stable and master.